speech/api/speech_streaming.py

```diff
@@ -1,11 +1,25 @@
 #!/usr/bin/python
+# Copyright (C) 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Sample that streams audio to the Google Cloud Speech API via GRPC."""
 
 import contextlib
 import re
 import threading
 
 from gcloud.credentials import get_credentials
-from google.cloud.speech.v1.cloud_speech_pb2 import *  # noqa
+from google.cloud.speech.v1 import cloud_speech_pb2 as pb
```

Comment:
I would rather you not alias it. If you are going to alias it, use `cloud_speech`.